src/main/java/com/amazonaws/kinesisvideo/java/service/JavaKinesisVideoServiceClient.java

```diff
@@ -166,6 +170,10 @@ public AWSCredentials getCredentials() {
                 try {
                     final KinesisVideoCredentials kinesisVideoCredentials = credentialsProvider.getCredentials();
 
+                    if (kinesisVideoCredentials == null) {
+                        throw new IllegalArgumentException("Credentials must not be null");
```

Comment:
It might be useful to differentiate these messages with the function name. Seems all are being set to the same string which can make debugging a little difficult.

Reply:
Updated

Reply:
Sorry. Should have been clear. I meant use function name wherever this message is being thrown. I see multiple instances of it.

Reply:
Sorry, I misunderstood as well. I've updated the classes to use log4j2 which wasn't the case earlier. log4j2 will now print the error messages with the class and function names everywhere